api/all/src/main/java/io/opentelemetry/api/trace/ImmutableSpanContext.java

```diff
@@ -19,12 +18,18 @@
           SpanId.getInvalid(),
           TraceFlags.getDefault(),
           TraceState.getDefault(),
-          /* remote= */ false);
+          /* remote= */ false,
+          /* valid= */ false);
 
   private static AutoValue_ImmutableSpanContext createInternal(
-      String traceId, String spanId, TraceFlags traceFlags, TraceState traceState, boolean remote) {
+      String traceId,
+      String spanId,
+      TraceFlags traceFlags,
+      TraceState traceState,
+      boolean remote,
+      boolean valid) {
     return new AutoValue_ImmutableSpanContext(
-        traceId, spanId, traceFlags, traceState, /* remote$= */ remote);
```

Comment:
You did not fix this from my previous comment :D

Reply:
I knew I was going to be in here hacking again, so I thought I'd just figure it out here. :)